### PR TITLE
Add Pyspark and Spark Sample Notebooks

### PR DESCRIPTION
**Pull request by aggFTW**

### CODE REVIEW

**Comment by aggFTW** [comment]
Closes #4!

File: examples/Pyspark Kernel.ipynb [hunk 1]
```diff
@@ -0,0 +1,686 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Features available with the new kernels\n",
+    "\n",
+    "By default Jupyter notebook comes with a `Python2` kernel. A kernel is a program that runs and interprets your code. sparkmagic provides two additional kernels that you can use with Jupyter. These are:\n",
```
Comment (msftristew) [inline examples/Pyspark Kernel.ipynb]: Jupyter can come with a Python2 kernel, but it can also come with a Python3 kernel

Comment (msftristew) [inline examples/Pyspark Kernel.ipynb]: Necessary to explain what a kernel is in these notebooks?

File: examples/Pyspark Kernel.ipynb [hunk 2]
```diff
@@ -0,0 +1,686 @@
...
+    "\n",
+    "1. **PySpark** (for applications written in Python). PySpark kernel exposes the Spark programming model to Python\n",
```
Comment (msftristew) [inline examples/Pyspark Kernel.ipynb]: > PySpark kernel exposes the Spark programming model to Python

Unnecessary sentence

File: examples/Pyspark Kernel.ipynb [hunk 3]
```diff
@@ -0,0 +1,686 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "------------\n",
+    "## How do I use the new kernels? \n",
+    "\n",
+    "1. Create a notebook with the new kernels. Click **New**, and then click **PySpark** or **Spark**. \n",
```
Comment (msftristew) [inline examples/Pyspark Kernel.ipynb]: Really basic tutorial -- who is the target audience for these example notebooks?

**Comment by msftristew** [comment]
So these example notebooks are the sample notebooks from the cluster copy-pasted here?  

I guess my only question would be whether the target audience for those notebooks is the same as the target audience for these notebooks, because if not, that might require changing the content.

File: examples/Pyspark Kernel.ipynb (hunk 3 above)
Comment (aggFTW) [inline examples/Pyspark Kernel.ipynb]: I think it's someone who wants to use Spark with Jupyter. Level of expertise greatly varies. Pros can skip these paragraphs while non-pros can read in depth.

**Comment by aggFTW** [comment]
@msftristew LGTY?

**Comment by msftristew** [comment]
ship it

**Pull request merged**